_includes/header.html

```diff
@@ -1,11 +1,16 @@
-<header>
+<header class="header">
+  <div class="header__brand">
+    <div class="logo header__logo">
+      <a href="{{ "/" | relative_url }}">
+        {% include /svg/logo_hm.html %}
+        {% include /svg/logo_hm_monogram.html %}
+      </a>
+    </div>
+    {% include /svg/icon_menu.html %}
+    {% include /svg/icon_close.html %}
+  </div>
 
-  <h1>
-    <a href="{{ "/" | relative_url }}">{{ site.title | escape }}</a>
-  </h1>
-
-  <nav>
+  <nav class="header__nav">
```

Comment:
this could be `header__nav nav`, might simplify sth

Reply:
also we might want to move this container to the `navigation.html` partial if there's no reason to do it here (and just leave `{% include navigation.html navigation=site.data.navigation %}`)